src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball;
+        for (int a = 0; a < 3; a++) {
+            ball = new Lottery().getRandomBall();
+            System.out.println(ball.toString());
```

Comment:
do not create each time new Lottery
```suggestion
            System.out.println(lottery.getRandomBall());
```